app/models/volunteer.rb

```diff
@@ -57,6 +63,10 @@ class Volunteer < User
                                      .order(:display_name)
                                  }
 
+  scope :birthday_next_month, -> {
+    where("EXTRACT(month from date_of_birth) = ?", DateTime.now.next_month.month)
```

Comment:
I tried, but could not come up with a way to do this without using EXTRACT

Reply:
Try this
```
where(date_of_birth: Date.today.beginning_of_month.next_month..Date.today.end_of_month.next_month)
```

Reply:
The problem is that this range will only include date_of_birth's that fall on the next month of this year.

Reply:
Good point. Yeah, the only solution I saw started to feel over-engineered. There's no user input in this, so that makes me more comfortable. Keep as is.